lib/models/os.ts

```diff
@@ -159,7 +159,7 @@ const getOsModel = function (
 		}
 		const vNormalized = v[0] === 'v' ? v.substring(1) : v;
 		if (!BALENAOS_VERSION_REGEX.test(vNormalized)) {
-			throw new Error(`Invalid semver version: ${v}`);
+			throw new Error(`Invalid balenaOS version format: ${v}`);
```

Comment:
This change goes back to a point I raised in another comment, whether we mean to validate any generic semver version or specifically balenaOS versions. This change aligns with the latter.